### PR TITLE
feat: Add Stackable vendor label to recommended labels

### PR DESCRIPTION
**Pull request by Techassi**
Part of https://github.com/stackabletech/issues/issues/188

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,8 +15,10 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 - Change Deref target of `KeyPrefix` and `KeyName` from `String` to `str` ([#725]).
+- Add Stackable vendor label to recommended labels ([#728]).
```
Comment (razvan) [inline CHANGELOG.md:18]:
```suggestion
- Add Stackable vendor label `stackable.tech/vendor: Stackable` to recommended labels ([#728]).
```

**Review by razvan — DISMISSED** [review]
One suggestion

**Comment by Techassi** [comment]
Added your suggestion.

**Review by razvan — APPROVED** [review]
sgtm

**Pull request merged**